pkg/apis/clusteroperator/types.go

```diff
@@ -525,6 +539,18 @@ const (
 	// on this machine set.
 	MachineSetInstalled MachineSetConditionType = "Installed"
 
+	// MachineSetComponentsInstalling is true if OpenShift components are being installed on
+	// this machine set.
+	MachineSetComponentsInstalling MachineSetConditionType = "Installing"
```

Comment:
I don't like that we have two different `MachineSetConditionType` constants that have the same value. Can we either (1) use the same const in both cases or (2) change the value for one of the consts. My preference would be (2) where, for example, "Installing" becomes "ComponentsInstalling" for `MachineSetComponentsInstalling`.

Reply:
@staebler thx, that was not intentional, rather careless copy/paste. Will fix.